_test/ensure-readmes-are-updated.sh

```diff
@@ -0,0 +1,36 @@
+if [ ! -x bin/configlet ]; then
+   echo "Improper configuration; configlet should exist in bin/ when this script is run"
+   echo "Ping a Rust track maintainer to fix this"
+   exit 1
+fi
+
+newline=$'\n  '
+
+missing_readmes=""
+wrong_readmes=""
+for exercise in $(git diff --name-only master..$(git rev-parse --abbrev-ref HEAD) | grep exercises/ | cut -d'/' -f2 -s | sort -fu); do
+   echo "Checking readme for $exercise"
+   readme_path="exercises/${exercise}/README.md"
+   if [ ! -f $readme_path ]; then
+      missing_readmes="$missing_readmes$newline$exercise"
+   else
+      existing_readme_checksum=$(md5sum $readme_path | cut -d' ' -f1)
+      # generate the new README
+      bin/configlet generate . --only "$exercise"
+      generated_readme_checksum=$(md5sum $readme_path | cut -d' ' -f1)
+
+      if [ $existing_readme_checksum != $generated_readme_checksum ]; then
```

Comment:
optional: I imagine you can check the exit code of `diff` rather than comparing the checksums against one another, if you find that easier.

Reply:
We don't actually have both files in place at the same time; the configlet command to regenerate it overwrites the existing README.md, if any. For this reason, I thought it easier to just keep the md5sum in memory and compare that.